app/controllers/concerns/activity_tracking.rb

```diff
@@ -2,29 +2,28 @@ module ActivityTracking
   protected
 
   def track_activity(trackable, action, user = current_user, project = nil)
-    project = current_project if project.nil? # current_project is set by ProjectScoped
     ActivityTrackingJob.perform_later(
       action: action.to_s,
-      project_id: project.id,
+      project_id: project ? project.id : nil,
       trackable_id: trackable.id,
       trackable_type: trackable.class.to_s,
       user_id: user.id
     )
   end
 
-  def track_created(trackable, user = current_user)
-    track_activity(trackable, :create, user)
+  def track_created(trackable, user: current_user, project: current_project)
```

Comment:
for parity, please add `track_assigned` here too

Reply:
Done!